### PR TITLE
GH-1455: n4js-cli postinstall: remove wget dependency in favor of usi…

### PR DESCRIPTION
**Pull request by dbo**
#1455

@mmews-n4 please

### CODE REVIEW

File: n4js-libs/packages/n4js-cli/README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ For more information about N4JS, please refer to the official [project home](htt
 When installing the package, you can replace the packaged `n4jsc.jar` using the environment variable `N4JSC_JAR` referencing
 either a filesystem path or a HTTP/HTTPS URL.
 
+See [download page](https://download.enfore.com/n4js/).
```
Comment (mmews-n4) [inline n4js-libs/packages/n4js-cli/README.md]: We might better link to the Eclipse download page. There we list both the Eclipse and the enfore download pages.

Comment (mmews-n4) [reply n4js-libs/packages/n4js-cli/README.md]: The page would be: https://www.eclipse.org/n4js/downloads.html

Comment (dbo) [reply n4js-libs/packages/n4js-cli/README.md]: but only the latter links directly to the jars, isn't it?

Comment (dbo) [reply n4js-libs/packages/n4js-cli/README.md]: or is there an eclipse jar download page?

**Pull request merged**